meta-rdk-mtk-bpir4/recipes-bsp/trusted-firmware-a/bootloader_prebuild.bb

```diff
@@ -0,0 +1,34 @@
+DESCRIPTION = "prebuild bl2.img and fip.bin inclusion"
+LICENSE = "RDK"
```

Comment:
How is the artifactory image licensed - is it RDK licensed software?  What headers on the files etc.?

Reply:
As discussed modified the recipe with new PR.
https://github.com/rdkcentral/meta-cmf-bananapi/pull/64